src/Assets/LightSkyDefense/Scripts/Enemies/PathFollower.cs

```diff
@@ -20,20 +20,28 @@ void Start()
         _path = GameManager.Instance.Path;
         transform.parent = null;
 
-        transform.position = _path.PathPoints[0].Position;
+        transform.position = _path.PathPoints[0];
         offsetTranslation = new Vector3(Random.Range(-_pathThickness, _pathThickness), Random.Range(-_pathThickness, _pathThickness), Random.Range(-_pathThickness, _pathThickness));
     }
 
     void FixedUpdate()
     {
-        Vector3 f = Vector3.MoveTowards(transform.position, _path.PathPoints[PathPointIndex].Position, 0.002f);
+        Vector3 f = Vector3.MoveTowards(transform.position, _path.PathPoints[PathPointIndex], 0.002f);
```

Comment:
What does `f` mean? This could be a proper variable name.

```suggestion
        var forward = Vector3.MoveTowards(transform.position, _path.PathPoints[PathPointIndex], 0.002f);
```